src/object/weak_block.cpp

```diff
@@ -203,6 +204,13 @@ WeakBlock::spreadHit()
   }
 }
 
+void
+WeakBlock::on_flip(float height)
+{
+  MovingSprite::on_flip(height);
+  FlipLevelTransformer::transform_flip(m_flip);
+}
+
```

Comment:
No, because that would flip all `MovingSprite`s, including those that shouldn't be flipped (like all gravity-affected objects) - badguys walking on their heads.